tests/e2e/e2e_test.go

```diff
@@ -285,10 +297,10 @@ func (ctx *testCtx) pingFromVPPClb(destAddress string) func() error {
 	}
 }
 
-func (ctx *testCtx) testConnection(fromMs, toMs, dstAddr, listenAddr string, port uint16, udp bool) error {
+/*func (ctx *testCtx) testConnection(fromMs, toMs, dstAddr, listenAddr string, port uint16, udp bool) error {
```

Comment:
fyi: preparing NAT and ACL tests, this method will be provided with it.

Reply:
Just wanted to avoid golangci bot from complaining that it's unused.